### DISCUSSION TITLE
Unifi Port Numbers to get pluging working?

### DISCUSSION THREAD

**Discussion by jkelly73** [briis/unifiprotect#296] [Q&A]
Hi guys,
I have been using this plugin for sometime with all my devices on the same VLAN. I recently moved my Homeassistant onto an IOT network / VLAN and secured it in the firewall (all via a UDM-Pro).

Now, When I try to re-install the plugin it cant connect to Unifi.  If I disable the 'drop all inter-VLAN traffic' rule (ie re-enable it), all works fine.

So, Im guessing that I need to create an 'allowable port' group and use this to 'allow' traffic between VLANs in the 'LAN IN' tab.  The question is, what are the ports?

Any thoughts?
Thx
Jon

**Comment by briis** [comment]
Just to make sure: You are talking about UniFi Protect, yes?

The only port numbers used here are 443, if you run on a UNVR or UDMP and 7443 if you run on a CloudKey+

**Reply by jkelly73** [reply]
Thanks.  Yes, my cameras are all Unifi cameras and all connect into a UNVR via a unifi (Pro-24) switch.  The LAN / VLANs are run via a UDM-Pro and its firewalls.

All my Unifi devices and UNVR are sat in my LAN.  My Homeassistant is sat in my IOT VLAN (20).  I have blocked all inter-vlan traffic from the VLAN but have unrestricetd traffic from LAN to all VLANs.

From the above (can provide more info if needs be) and your initial response, am I right in thinking that I should only need to allow Port 443 through? from VLAN to LAN?  I think this would be a rule in the "LAN IN" in the UDM-Pro Firewall.

Any pointers would be grately appreciated.

Thx
Jon

**Comment by briis** [comment]
I am afraid I am not a network expert, and I am not sure how to setup LAN -> VLAN so that it will work. My I suggest you ask in the community forum, I know there are many people in there who have a similar setup like yours.

**Comment by AngellusMortis** [comment]
You want the following ports between your UDM Pro / UNVR and Home Assistant:

* 80/443 (tcp): Main HTTPS/WSS
* 7447 (tcp?) (integration <= v0.9,1): RTSP for cameras
* 7441 (tcp?) (integration >= v0.10.0): RTSPS for cameras

You also want the following ports between Home Assistant and _your cameras_:

* 80/443 (tcp): HTTP - only if you are using anonymous snapshots
* 7004 (udp): Talkback settings - we are not currently using this, but eventually will be to play audio through your camera speakers.

This _may_ not a  comprehensive list, but it should be all of the ports the library itself uses.
